README.md

```diff
@@ -8,52 +8,60 @@
 
 An example implementation of the [NLP Sandbox Date Annotator].
 
-## Specification
+## Overview
 
-- Implements the [NLP Sandbox Date Annotator OpenAPI specification]
-- Annotates date strings in clinical notes using simple regular expressions
+This repository provides a Python-Flask implementation of the [NLP Sandbox Date
+Annotator]. The Date Annotator is one of the first NLP Tools that can be
+benchmarked on nlpsandbox.io. A Date Annotator takes as input one clinical notes
+as well as information about the patient and outputs a list of predicted date
+annotations found in the clinical note.
 
-NLP Sandbox API implemented:
+- Date Annotator API version: 0.3.1
+- Tool version: 0.3.2
+- Docker image: [nlpsandbox/date-annotator-example]
 
-| API | Type | Version |
-|---|---|---|
-| [Date Annotator](/openapi/date-annotator) | `nlpsandbox:date-annotator` | 0.3.2 |
+## Model
 
-## Usage
+This NLP tool relies on simple regular expressions to identify the location of
+date strings in a clinical note.
+
+This implementation is provided as an example that Developers can use to quick
+start the development of a new model by creating a repository from this [GitHub
+template] (see below). The Docker image built and published automatically by the
+CI/CD workflow of this repository can be submitted "as it is" to the [NLP
+Sandbox] to benchmark its performance if you wish to try submitting first (don't
+expect a high performance!).
 
-[![Gitpod](https://img.shields.io/badge/Gitpod-Open_in_Gitpod-blue?color=0273b3&labelColor=555555&logoColor=ffffff&style=for-the-badge&logo=gitpod&label=)](https://gitpod.io/#https://github.com/nlpsandbox/date-annotator-example)
+## Usage
 
 ### Running with Docker
 
 The command below starts the Date Annotator locally.
 
-    docker-compose up
+    docker-compose up --build
```

Comment:
Maybe add a comment on how to stop the run?

"You can stop the container run with `^C`, followed by `docker-compose down`."

Reply:
Added the sentence (I used `Ctrl+C`)